app/src/main/resources/application.properties

```diff
@@ -9,11 +9,13 @@ quarkus.datasource.db-kind=h2
 quarkus.hibernate-orm.database.generation=drop-and-create
 
 # vault url
-%prod.quarkus.vault.url=http://localhost:8200
+%prod.quarkus.vault.url=http://vault.65.108.212.158.nip.io
+%dev.quarkus.vault.url=http://vault.127.0.0.1.nip.io
 
 # vault authentication
-%prod.quarkus.vault.authentication.userpass.username=bob
-%prod.quarkus.vault.authentication.userpass.password=sinclair
+quarkus.vault.authentication.userpass.username=bob
+quarkus.vault.authentication.userpass.password=sinclair
+
+%prod.quarkus.vault.kv-secret-engine-version=1
+%prod.quarkus.vault.kv-secret-engine-mount-path=kv
```

Comment:
I think that the mount path should be `secret` and not `kv` as that should correspond to this command that you can issue using the vault client `vault kv put -mount=secret primaza/hello target=world`

Reply:
According with what is configured by [vault installation script](https://github.com/halkyonio/primaza-poc/blob/a8d93e2a4a89156e4c5d25d85347a72742d139ae/scripts/vault.sh#L12) it needs to be `kv`.